js/bs-init.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function(){
+	AOS.init();
+});
```

Comment:
A three line JS file? Might as well inline this in the HTML template and safe us that additional HTTP call to fetch this.